tasks/base_tasks.py

```diff
@@ -747,6 +750,21 @@ def mark_done(self):
             table=self.output().table))
 
 
+class LoadPostgresFromZipFile(TempTableTask):
+
+    def load_from_zipfile(self, zipfile):
+        shell('gunzip {zipfile} -c | grep -v default_tablespace | psql'.format(
+            zipfile=zipfile))
+        self.mark_done()
+
+    def mark_done(self):
```

Comment:
Could you please explain me what's this for? I guess that it's Luigi-related, but...

Reply:
Luigi uses the target in the `output` function to know when the task can be marked as `DONE`. In this case, the schema is loaded from a URL (or a zip file after this PR) that contains all the SQL statements needed to create the schema, tables and insert the data instead of being just a plain CSV file, so we can't control the table/schema name and we create a fake table from the  output to know when all the data has been loaded.
It's far from being perfect but as it involved Tiger/ACS changed I have preferred not to change it ATM.

Reply:
Ok, but how come the table must be _recreated_ on `mark_done`? If that's just a defensive measure you could use `IF NOT EXISTS` instead.

In addition, maybe adding the explanation as a comment might be nice, as this is not obvious for a reader.

Reply:
It's a small table with one row that stores the creation time of the table. If we use `IF NOT EXISTS` we would need to do an `UPDATE` or `INSERT` afterwards. This `LoadPostgresFromZipFile` just replicates the existing `LoadPostgresFromURL` but using an existing file instead of downloading it.
I'm adding a comment to make it clear.